zhaquirks/ikea/__init__.py

```diff
@@ -87,6 +181,34 @@ class PowerConfiguration2AAACluster(DoublingPowerConfigurationCluster):
     }
 
 
+class PowerConfigurationND2AAACluster(PowerConfigurationCluster):
```

Comment:
Just noticed whilst working on the [other PR](https://github.com/zigpy/zha-device-handlers/pull/2284):
You're extending the custom [`PowerConfigurationCluster`](https://github.com/zigpy/zha-device-handlers/blob/fa60b9d23a3e94b5a164e6aad1f24f4134544daa/zhaquirks/__init__.py#L195-L229) instead of `PowerConfiguration` here.
IMO, that cluster is named badly, as it doesn't mention what it's actually doing.
It seems like that cluster is used for devices which only report their battery voltage (and not percentage).
All IKEA devices do also report percentage, so I'm guessing that's supposed to be just `PowerConfiguration`?

Reply:
Its coming from the original implementation and i think the the naming was the same as was doing with the LightLinkCluster.
Change if you like.

Reply:
Wasn't talking about the naming, but there's a difference between `PowerConfiguration` and `PowerConfigurationCluster`.

The previous implementation for IKEA devices always used `DoublingPowerConfigurationCluster` which inherited from the ZCL `PowerConfiguration` (and not the remapping `PowerConfigurationCluster`).

So there's a difference in behavior, as your current code would ignore the battery percentage value, as it's remapped from the battery voltage. I don't think that's wanted for these IKEA devices, as they properly report percentage, right?

Reply:
I think you is being right then i was copy the ground for the quirk from Styrbar (4btn) remote that was changed from doubling to no doubling and then its is also having the wrong class for the battery.
Can you fixing to the right class after you have doing the large rework of the IKEA INIT ??

I need my first KAFFEE for the day ;-)

Reply:
Yep, will rebase this PR and fix that minor issue once the battery PR is merged.